toolkit/src/components/cut/filter-bar/search.hbs

```diff
@@ -18,6 +21,9 @@
   <Hds::Form::TextInput::Base
     @type='search'
     @value={{@search}}
+    @width={{or @width '196px'}}
```

Comment:
I thought about setting this in a component property but seemed like a bit much for a single property. idk, happy to hear thoughts.